scripts/bundle-image-pickup.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright (c) 2022 Red Hat, Inc.
+###############################################################################
+
+####################
+## COLORS
+####################
+CYAN="\033[0;36m"
+GREEN="\033[0;32m"
+PURPLE="\033[0;35m"
+RED="\033[0;31m"
+YELLOW="\033[0;33m"
+NC="\033[0m"
+
+log_color () {
+  case $1 in
+    cyan)
+      echo -e "${CYAN}$2 ${NC}"$3
+    ;;
+    green)
+      echo -e "${GREEN}$2 ${NC}"$3
+    ;;
+    purple)
+      echo -e "${PURPLE}$2 ${NC}"$3
+    ;;
+    red)
+      echo -e "${RED}$2 ${NC}"$3
+    ;;
+    yellow)
+      echo -e "${YELLOW}$2 ${NC}"$3
+    ;;
+  esac
+}
+
+####################
+## ENV VARIABLES
+####################
+ORG=${ORG:-"stolostron"}
+PIPELINE_REPO=${PIPELINE_REPO:-"pipeline"}
+RELEASE_BRANCH=${RELEASE_BRANCH:-"2.7-integration"}
+
+####################
+## PATHS (I.E DIR, FILES, ETC)
+####################
+PIPELINE_MANIFEST_FILEPATH=${PIPELINE_MANIFEST_FILEPATH:-"$PIPELINE_REPO/manifest.json"}
+OPERATOR_CSV_FILEPATH=${OPERATOR_CSV_FILEPATH:-"bundle/manifests/search-v2-operator.clusterserviceversion.yaml"}
+
+OPERATOR_CONTAINER_PATH=${OPERATOR_CONTAINER_PATH:-".spec.install.spec.deployments[0].spec.template.spec.containers[1]"}
+OPERATOR_ENV_PATH=${OPERATOR_ENV_PATH:-"$OPERATOR_CONTAINER_PATH.env[].values"}
+OPERATOR_IMAGE_PATH=${OPERATOR_IMAGE_PATH:-"$OPERATOR_CONTAINER_PATH.images"}
+
+####################
+## IMAGE VARIABLES
+####################
+IMG_REGISTRY=${IMG_REGISTRY:-"quay.io/$ORG"}
+
+# DEFAULT IMAGES
+DEFAULT_TAG=${DEFAULT_TAG:-"2.6.0-SNAPSHOT-2022-08-08-20-23-21"}
+DEFAULT_OPERATOR_TAG=${DEFAULT_OPERATOR_TAG:-"2.6"}
+
+# SEARCH V1 IMAGES
+DEFAULT_COLLECTOR_IMAGE=$IMG_REGISTRY/search-collector:$DEFAULT_TAG
+
+# SEARCH-V2 IMAGES
```

Comment:
Collector will become part of V2.
```suggestion
DEFAULT_COLLECTOR_IMAGE=$IMG_REGISTRY/search-collector:$DEFAULT_TAG
```

Reply:
Updated this section to just contain `SEARCH IMAGES`.